### DISCUSSION TITLE
The new auto-complete only works for the first property in query

### DISCUSSION THREAD

**Discussion by CrossBound** [datalust/seq-tickets#2138] [General]
I like the new auto-complete introduced in 2023.4, but not sure if it's just me or not but when I search the auto-complete only works for the first property I work with. If I try to do a compound search filter (combining filters with "AND" or "OR") auto-complete doesn't work for secondary filter clauses

**Comment by CrossBound** [comment]
It looks like after the first clause, it will autocomplete for system properties (like "and", "or", "@timstamp", etc.) but not user properties attached to a log event.

**Comment by KodrAus** [comment]
Thanks @CrossBound! You're totally right, I've logged a bug as an issue for this. We can add any more details over there: https://github.com/datalust/seq-tickets/issues/2139
